README.md

```diff
@@ -21,6 +21,12 @@ Please see [Contributing](CONTRIBUTING.md) for how to contribute to this project
 - [**Qt6**](https://www.qt.io/)
 - [**libusb-1.0**](http://libusb.info/)
 
+For developers on Ubuntu or other Debian-based distributions, appropriate versions of several of the required packages are unlikely to be installed by default.
```

Comment:
Nice idea, documenting the packages might be helpful for other distros too :+1:.

Interestingly, you didn't need additional packages `libglx-dev` and `libgl1-mesa-dev`, needed by the CI build: https://github.com/offa/plug/blob/master/script/ci_setup.sh#L7

Did you have those on your system already?

Reply:
A first version: 95fafcbbeb23f576b0be68a1ac8ebf649dee57a8

Reply:
Both of those packages are installed on my system - I was only able to list the packages which were missing at the time I started work on this change.